source/blog/2021-03-07-this-month-in-solidus-february-2021.html.erb.markdown

```diff
@@ -0,0 +1,69 @@
+community, but we felt it was time we gave it a proper full-time maintainer, and
+[Nebulab](https://nebulab.it) stepped in to fill the gap.
+
+[Marc Busqué](https://github.com/waiting-for-dev) is a senior Ruby developer based in beautiful
+[Cambrils, Spain](https://en.wikipedia.org/wiki/Cambrils). He has an impressive track record of
+contributing to OSS, having worked on the renowned [dry-rb](https://github.com/dry-rb) ecosystem.
+
+Marc recently joined the Nebulab team, where he will work with the Core Team on the maintenance and
+evolution  of Solidus, splitting his time between keeping the platform in good shape and pushing it
+forward with his technical expertise.
+
+Welcome aboard Marc, and best of luck!
+
+P.S. Pretty soon, Marc should be in good company... Stay tuned!
+
+### Backend UX Audit 🎨
+
+Last Thursday, the stakeholders have approved a vote to engage [Baymard](https://baymard.com/) for
+a UX audit of the Solidus backend.
+
+We are _extremely_ excited about this: Baymard is a world-class UX institute that has worked with
+some of the best players in the industry, and we're looking forward to using their expertise to
+improve the Solidus backend experience.
+
+Baymard can use all the help they can get, so if you have any feedback to share about the Solidus
+backend that you think would be beneficial for this audit, don't be shy about stepping forward! You
+can reach out to any Core Team member on Slack, who will direct you to the proper channels.
+
+### ShipStation API Integration 📦
+
+[`solidus_shipstation`](https://github.com/solidusio-contrib/solidus_shipstation) was recently
+overhauled to allow users to benefit from the new [ShipStation API](https://www.shipstation.com/docs/api/),
+which will eventually supersede the legacy [XML integration](https://help.shipstation.com/hc/en-us/articles/360025856192-Custom-Store-Development-Guide).
+
+The new API integration provides several advantages over the previous one: it is much more flexible,
+providing you with greater granularity of how you import and handle your orders in ShipStation, and
+much more performant, allowing you to sync up to 4000 orders/minute from your Solidus store.
+
+Most importantly, we built the new integration so that switching from the legacy XML format is
+completely painless, so go ahead and check it out right now!
+
+### In The Works: Certified TaxJar Integration 💸
+
+[SuperGood's TaxJar integration](https://github.com/SuperGoodSoft/solidus_taxjar) has always been
+amazing, but it's about to get _so much better!_
+
+The SuperGood team is working towards getting the extension [fully certified](https://developers.taxjar.com/integrations/certification-guidelines/),
+which means they will add a ton of new features in the process.
+
+They could definitely use some help, so if you have worked with TaxJar in the past and want to
+contribute, hop on GitHub!
+
+_<p style="text-align: right">Aaaand that's it for now. See you next month! 🚀</p>_
```

Comment:
What about adding some words about the preparation for Solidus 3.0 and the Hackathon?

### Solidus 3 is getting closer :shipit: 

We are working hard to have everything set for the release of Solidus 3.0 and last month we organized a Hackathon to have the community extensions ready for the next major bump. 

We've got **23 extensions** ready in a single day and well... this is just amazing. Thanks, community! 
We still have some extensions to prepare though so if you want to help, there's still time: just follow the instructions in this [GitHub Discussion](https://github.com/solidusio/solidus/discussions/3911).

Reply:
@kennyadsl done!